server/src/main/java/org/apache/druid/server/coordinator/duty/CompactSegments.java

```diff
@@ -175,7 +176,7 @@ public DruidCoordinatorRuntimeParams run(DruidCoordinatorRuntimeParams params)
     // Skip all the intervals locked by higher priority tasks for each datasource
     // This must be done after the invalid compaction tasks are cancelled
     // in the loop above so that their intervals are not considered locked
-    getLockedIntervalsToSkip(compactionConfigList).forEach(
+    getLockedIntervalsV2(compactionConfigList).forEach(
```

Comment:
The name of this method need not change. V1/V2 makes sense only in the context of the API.

Reply:
Done